### ISSUE TITLE
Remove CSP config tests

### ISSUE THREAD

**Issue by muffinresearch** [mozilla/addons-frontend#2449]
Now that the CSP middleware + config are tested together in a unit test I think we can drop the standalone CSP config tests.

**Comment by kumar303** [comment]
Oh, yeah, I want this because currently they might fail if you have a `config/local-production.js` file with conflicting values.

**Comment by muffinresearch** [comment]
@kumar303 I don't think removing this would fix that alone since the CSP middleware tests also look at config. 

We should have a brainstorm on some possible solutions for that.

**Comment by kumar303** [comment]
Yeah, good call. I had changed the check for this to allow `local-production.js` config files but I reverted that in https://github.com/mozilla/addons-frontend/commit/54d18a826dedaf0ecdf30ce329c2d14490b7cc48 It will now give you an error if you have one of those configs which should alleviate confusion.

**Comment by atiqueahmedziad** [comment]
@muffinresearch the links you have mentioned is not invalid now. Is the bug still valid?

**Comment by kumar303** [comment]
I think it might still be valid. Here are the updated links:

* standalone CSP tests: https://github.com/mozilla/addons-frontend/blob/master/tests/unit/TestCSPConfig.js
* CSP middleware tests: https://github.com/mozilla/addons-frontend/blob/master/tests/unit/core/middleware/test_cspMiddleware.js

**Comment by SeanPrashad** [comment]
* standalone CSP tests: https://github.com/mozilla/addons-frontend/blob/master/tests/unit/test_cspConfig.js

(New link)

**Comment by SeanPrashad** [comment]
I'm gunna take a crack at this one! All of the tests in the old standalone file aren't found in the new middleware one. My assumption is that we should transfer them over.. but at the same time, they might be implicitly covered in the newer middleware test suite? 🤔

**Comment by SeanPrashad** [comment]
Hey @kumar303 and @muffinresearch  - just wanted to double check before I begin and see if this bug is still valid and if we will still need to consider transferring over the old standalone tests?

**Comment by muffinresearch** [comment]
As long as any changes don't reduce the relevant coverage then I don't see why not.
